lib/phobos.rb

```diff
@@ -9,16 +9,20 @@
 require 'exponential_backoff'
 require 'kafka'
 require 'logging'
+require 'erb'
 
 require 'phobos/deep_struct'
 require 'phobos/version'
 require 'phobos/instrumentation'
 require 'phobos/errors'
 require 'phobos/listener'
+require 'phobos/actions/process_batch'
+require 'phobos/actions/process_message'
 require 'phobos/producer'
 require 'phobos/handler'
 require 'phobos/echo_handler'
 require 'phobos/executor'
+require 'phobos/spec_helper'
```

Comment:
why? 🤔

Reply:
this is a convenient spec helper to help test consumer with minimal setup required. it is not `spec/spec_helper.rb` :)